mozilla_django_oidc_db/setupconfig/bootstrap.py

```diff
@@ -0,0 +1,80 @@
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.contrib.sessions.middleware import SessionMiddleware
+from django.test import RequestFactory
+
+from django_setup_configuration.config_settings import ConfigSettings
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import ConfigurationRunFailed, SelfTestFailed
+
+from ..forms import OIDCSetupConfigForm
+from ..models import OpenIDConnectConfig
+from ..views import OIDCAuthenticationRequestView
+
+
+class AdminOIDCConfigurationStep(BaseConfigurationStep):
+    """
+    Configure admin login via OpenID Connect
+    """
+
+    verbose_name = "Configuration for admin login via OpenID Connect"
```

Comment:
can you mark this translatable? Or does that not make any sense?

Reply:
Marked as translatable but this is only displayed when running the command.